pkg/templates/notificationtemplates/notification_generator_test.go

```diff
@@ -36,9 +36,18 @@ func TestGetNotificationTemplate(t *testing.T) {
 			require.NoError(t, err)
 			require.NotNil(t, template)
 			require.NotNil(t, template["userprovisioned"])
```

Comment:
It can never be nil, can it?
```suggestion
			require.NotEmpty(t, template["userprovisioned"])
```

Reply:
fixed as part of: 43b2953aa3f7922169954600dd7d4cabfbe0a2ac